App.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {StyleSheet, Text, View, Image, TouchableOpacity} from 'react-native';
+
+const App = () => {
+  return (
+    <View style={styles.homePage}>
+      <View style={styles.logoContainer}>
+        <Image source={require('./assets/logo/logo.png')}></Image>
+      </View>
+      <View style={styles.tagLineContainer}>
```

Comment:
Use proper naming convention for container and its child styles,
eg. It should be either `titleContainer` and `title`
or `taglineContainer` and `tagline`